src/js/components/Menu/Menu.js

```diff
@@ -70,55 +70,45 @@ const Menu = props => {
   const iconColor = normalizeColor('control', theme);
   const align = dropProps.align || dropAlign;
   const buttonRefs = {};
+  const NONE = 'none';
+  const tab = 9;
 
-  // to track if focus is within the menu items, see menuOnFocus/menuOnBlur
-  let timeoutID;
-
-  const [activeItemIndex, setActiveItemIndex] = useState(-1);
+  const [activeItemIndex, setActiveItemIndex] = useState(NONE);
   const [isOpen, setOpen] = useState(open || false);
-  const [isManagingFocus, setManagingFocus] = useState(false);
-
-  useEffect(() => {
-    if (isOpen) {
-      setManagingFocus(true);
-      if (isManagingFocus) {
-        if (align.top === 'bottom') {
-          buttonRefs[0].focus();
-          setActiveItemIndex(0);
-        } else {
-          buttonRefs[items.length].focus();
-          setActiveItemIndex(-1);
-        }
-      }
-    }
-  }, [isManagingFocus]);
 
   const onDropClose = () => {
-    setActiveItemIndex(-1);
+    setActiveItemIndex(NONE);
```

Comment:
why do we need this line at all now? If the Menu Button itself shouldn't have hoverindication than this line may be redundant.

Reply:
Good question. In this component, the active item index is changed based on what item has focus (not hover). So, it can be affected when the user uses keyboard to move through menu items. Hover indication doesn't actually change the active item index. What this line does is ensures that the active item index returns to its default value when the menu closes.